.github/workflows/release.yml

```
@@ -41,12 +42,51 @@ jobs:
       - name: Make
         run: pyinstaller -i icon.ico --clean --noconsole --add-binary "icon.ico;." --onefile main.py
 
-      - name: Upload Release Asset
+      - name: Upload to Release Asset
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ needs.release-please.outputs.upload_url }}
-          asset_path: dist\main.exe
-          asset_name: CurseForgeModpackDownloader-${{ needs.release-please.outputs.tag_name }}.exe
+          asset_path: ./dist/main.exe
+          asset_name: CurseForgeModpackDownloader-${{ needs.release-please.outputs.tag_name }}-Windows.exe
           asset_content_type: application/vnd.microsoft.portable-executable
+
+  linuxbuild:
+    runs-on: ubuntu-latest
+    name: Pack on Linux and Upload Release
+    needs: release-please
+    if: ${{ needs.release-please.outputs.release_created }}
+
+    steps:
+      - name: Check Out
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Make
+        run: pyinstaller -i icon.ico --clean --noconsole --onefile main.py
+
+      - name: Generate Start Up Script
+        run: cd dist && echo "#!/bin/sh" >> start.sh && echo "export SSL_CERT_DIR=/etc/ssl/certs" >> start.sh && echo "./main" >> start.sh
+
+      - name: Compress Application
+        run: cd dist && tar -cvf pack.tar.gz start.sh main
+
+      - name: Upload to Release
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ needs.release-please.outputs.upload_url }}
+          asset_path: ./dist/pack.tar.gz
+          asset_name: CurseForgeModpackDownloader-${{ needs.release-please.outputs.tag_name }}-Linux.tar.gz
+          asset_content_type: application/x-compressed-tar
```

https://en.wikipedia.org/wiki/List_of_archive_formats#:~:text=txz%2C%20.tar.zst-,application/x%2Dgtar,-tar%20with%20gzip
```suggestion
          asset_content_type: application/x-gtar
```